src/Raven.Studio/typescript/viewmodels/database/tasks/editSubscriptionTask.ts

```diff
@@ -120,10 +166,39 @@ class editSubscriptionTask extends viewModelBase {
 
     compositionComplete() {
         super.compositionComplete();
+        
+        document.getElementById('taskName').focus();
+    }
 
-        $('.edit-subscription-task [data-toggle="tooltip"]').tooltip();
+    attached() {
+        super.attached();
+
+        this.editedSubscription().startingChangeVector.subscribe(() => {
+            this.documentIDForUserDefinedChangeVector(null);
+            this.triedToFindDocumentForUserDefinedChangeVectorButFailed(false);
+        });
         
-        document.getElementById('taskName').focus(); 
+        popoverUtils.longWithHover($("#last-change-vector-acknowledged"),
+            {
+                content: this.editedSubscription().lastChangeVectorAcknowledgedHtml(), 
+                placement: "top"
+            });
+
+        popoverUtils.longWithHover($("#next-batch-starting-point"),
+            {
+                content: this.editedSubscription().changeVectorForNextBatchStartingPointHtml(),
+                placement: "top"
+            });
+
+        popoverUtils.longWithHover($(".no-matching-document-found"),
```

Comment:
sometimes you use id selector but it this case it is class selector. Is it typo?

Reply:
It is not a typo.
This class is used in 2 places in the html.

Reply:
ok great